test/extensions/filters/http/adaptive_concurrency/concurrency_controller/gradient_controller_test.cc

```diff
@@ -462,6 +483,51 @@ TEST_F(GradientControllerTest, TimerAccuracyTest) {
   concurrency_update_interval:
     nanos: 123000000 # 123ms
 min_rtt_calc_params:
+  jitter:
+    value: 10.0
+  interval:
+    seconds: 100
+  request_count: 5
+)EOF";
+
+  // Verify the configuration affects the timers that are kicked off.
+  NiceMock<Event::MockDispatcher> fake_dispatcher;
+  auto sample_timer = new NiceMock<Event::MockTimer>;
+  auto rtt_timer = new NiceMock<Event::MockTimer>;
+
+  // Expect the sample timer to trigger start immediately upon controller creation.
+  EXPECT_CALL(fake_dispatcher, createTimer_(_))
+      .Times(2)
+      .WillOnce(Return(rtt_timer))
+      .WillOnce(Return(sample_timer));
+  EXPECT_CALL(*sample_timer, enableTimer(std::chrono::milliseconds(123), _));
+  auto controller = std::make_shared<GradientController>(makeConfig(yaml), fake_dispatcher,
+                                                         runtime_, "test_prefix.", stats_, random_);
+
+  // Set the minRTT- this will trigger the timer for the next minRTT calculation.
+
+  // Let's make sure the jitter value can't exceed the configured percentage as well by returning a
+  // random value > 10% of the interval.
+  EXPECT_CALL(random_, random()).WillOnce(Return(15000));
+  EXPECT_CALL(*rtt_timer, enableTimer(std::chrono::milliseconds(105000), _));
+  for (int i = 0; i < 6; ++i) {
+    tryForward(controller, true);
+    controller->recordLatencySample(std::chrono::milliseconds(5));
+  }
+}
+
+TEST_F(GradientControllerTest, TimerAccuracyTestNoJitter) {
+  const std::string yaml = R"EOF(
+sample_aggregate_percentile:
+  value: 50
+concurrency_limit_params:
+  max_gradient: 3.0
+  max_concurrency_limit:
+  concurrency_update_interval:
+    nanos: 123000000 # 123ms
```

Comment:
nit: can you just switch this over to the other format (`0.123s`), same elsewhere.